pom.xml

```diff
@@ -132,7 +132,7 @@
     <grpc.version>1.28.1</grpc.version>
     <netty.version>4.1.45.Final</netty.version>
     <rocksdb.version>5.15.10</rocksdb.version>
-    <hadoop.version>2.7.3</hadoop.version>
+    <hadoop.version>3.3.0</hadoop.version>
```

Comment:
Does this cause any issues with the Hadoop compatible API? Is the FileSystem interface backward compatible?